nss-day-cohort-60/rare-client-tenacious-tators
# Evn Button and Color Styling

# Description of Proposed Changes
Cohesive buttons and font colors
---


# Steps to Test

1. Fetch the `EVN-styling` branch and switch to it
2. Be sure you pull origin EVN-styling once on the branch to make sure you are up to date
3. Check anywhere and everywhere there should be a button (even login and register) and make sure it's round, and has some styling
4. Make sure all of the font colors are appeasing and don't clash with any of your edits
